src/ui/components/ShowMoreCard/ShowMoreCard.scss

```diff
@@ -2,7 +2,9 @@
 
 .ShowMoreCard-contents {
   &::after {
-    background: linear-gradient(transparent, $base-color);
+    // rgba(255,255,255,0) instead of transparent prevents
+    // grey line in Safari see #2865
+    background: linear-gradient(rgba(255,255,255,0), $base-color);
```

Comment:
I think the linter doesn't like the lack of spaces here. Also to prevent using non-variable colours you could do `transparent($base-color, 1)` or whatever the SCSS was, but that's maybe superfluous.